docs/scos/dev/front-end-development/202307.0/oryx/building-applications/styling/oryx-responsive-design.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Oryx's Responsive Design system"
+description: Learn how to leverage Oryx's Responsive Design system to create stunning and user-friendly websites and applications that seamlessly adapt to different screen sizes and devices.
+last_updated: October 3rd, 2023
+template: concept-topic-template
+---
+
+In today's digital landscape, it is crucial to create websites and applications that adapt seamlessly to different screen sizes and devices. Oryx's Responsive Design system empowers developers to implement designs for a variety of screen sizes and layouts without in a convenient way.
+
+Oryx's Responsive Design System provides various screen sizes which you can redefine when needed. The associated CSS styles, that are linked to the screen sizes will adapt automatically to the configuration that your provide for your application. This makes the Responsive Design System useful for different types of applications. For example, if you need to build a business application where users will have a large screen available, you can optimize the layout of your application to benefit from all the available space.
+
+The responsive design system works closely with other styling techniques in Oryx, such as layouts, [design tokens](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-applications/styling/oryx-design-tokens.html), themes and [typography](/docs/scos/dev/front-end-development/{{page.version}}/oryx/building-applications/styling/oryx-typography.html) and play an important role in some of the design system components, such as media and images.
+
+<!-- Add link to layout docs when they're published -->
+<!-- Add link to themes docs when they're published -->
+<!-- Add link to (design system) components docs when they're published -->
+
+## Screen sizes
+
+Oryx is designed to render applications and their components on a variety of devices, from smartphones and tablets to desktop monitors or even smartwatches. The Design System is not connected to the various devices, but the screen sizes associated to those devices. Screen sizes are not one to one related to devices. A small screen design, for example, is not only used for mobile devices, but can also be used on large screens in physical stores.
+
+Oryx provides five t-shirt size values, that are given by a generic `size` enumeration, that can be imported from the [utilities package](https://www.npmjs.com/package/@spryker-oryx/utilities).
+
+| Screen      | Code      | Example devices                  |
+| ----------- | --------- | -------------------------------- |
+| Extra small | `Size.xs` | Small phone, Smart Watch         |
+| Small       | `Size.sm` | Smart phone, instore application |
+| Medium      | `Size.md` | Tablet                           |
+| Large       | `Size.lg` | Desktop                          |
+| Extra large | `Size.xl` | Wide screen                      |
```

Comment:
```suggestion
| Extra small | `Size.Xs` | Small phone, Smart Watch         |
| Small       | `Size.Sm` | Smart phone, instore application |
| Medium      | `Size.Md` | Tablet                           |
| Large       | `Size.Lg` | Desktop                          |
| Extra large | `Size.Xl` | Wide screen                      |
```

Reply:
Maybe better to add additional column for the enumerable value - Enum breakpoint for example, and the Code fill with t-shirt like sizes? ie:
```
| Screen        | Code | Enum breakpoint | Example devices                  |
| ----------- | ----- |------------ | --------------------------- |
| Extra small | `xs`  | `Size.Xs`     | Small phone, Smart Watch  |
```